Source/DafnyCore/Backends/Java/JavaCodeGenerator.cs

```diff
@@ -884,7 +884,7 @@ protected override string TypeName_UDT(string fullCompileName, List<TypeParamete
     // We write an extern class as a base class that the actual extern class
     // needs to extend, so the extern methods and functions need to be abstract
     // in the base class
-    protected override bool IncludeExternMembers { get => true; }
+    protected override bool IncludeImportedMembers => true;
```

Comment:
Consider using "ExternImported" rather than just "Imported" throughout, only because if we also start saying just "Exported" for the other direction, it will be easily confused with export sets.